### PR TITLE
feat: Backport cms 4.0.x - Django 3.2 with Python 3.8 & 3.9 support, including update tooling

### PR DESCRIPTION
**Pull request by yakky**
# Description

Merge code from #70 with updated develop

## References

Fix #70 

# Checklist

* [x] I have read the [contribution guide](https://djangocms-page-sitemap.readthedocs.io/en/latest/contributing.html)
* [x] Code lint checked via `inv lint`
* [x] ``changes`` file included (see [docs](https://djangocms-page-sitemap.readthedocs.io/en/latest/contributing.html#pull-request-guidelines))
* [x] Usage documentation added in case of new features
* [x] Tests added

### CODE REVIEW

**Comment by yakky** [comment]
@Aiky30 can you review this port of #70? I have merged with latest update from develop to make sure the two branches don't drift apart

I'll make a release to divio cloud with the current state of this branch and a beta one after merge

File: AUTHORS.rst [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 Development Lead
 ----------------
 
-* Iacopo Spalletti <i.spalletti@nephila.it>
+* Iacopo Spalletti <i.spalletti@nephila.digital>
 
 Contributors
 ------------
 
 * Anton Egorov
```
Comment (Aiky30) [inline AUTHORS.rst:13]: Not sure if this is auto populated. 
```suggestion
* Aiky30
* Anton Egorov
```

File: requirements-test.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ coveralls
 flake8>=2.1.0
 tox>=2.0
 django-app-helper
+https://github.com/Aiky30/djangocms-versioning/tarball/feature/django-32-compat#egg=djangocms-versioning
```
Comment (Aiky30) [inline requirements-test.txt:6]:
```suggestion
https://github.com/django-cms/djangocms-versioning/tarball/master#egg=djangocms-versioning
```

This was an old pin before 3.2 changes were merged.

**Review by Aiky30 — CHANGES_REQUESTED** [review]
Looks good, the only minor thing is the pinning of djangocms-verisoning. I updated the authors file but that could be auto populated so may undo any changes. 

Also, great to have you back around @yakky, we genuinely missed you man.

**Comment by yakky** [comment]
Replaced by #83 which includes the whole set of django CMS 4 changes

**Pull request closed without merging**